### PR TITLE
Ip graphs

### LINKED ISSUE #43: Graph visualizations for PCAP data
Add to tables page or include separate page to aggregate interesting table data

- [x] % of packets are Malicious
- [x] % of Protocols use 
- [x]  % of Incoming/outgoing traffic

### PR DESCRIPTION
**Pull request by Erik-Handeland**
# Description

Added graphs to enhance table visuals

# Checklist:

- [ ] Code follows the style guidelines
- [ ] Code commented in relevant places
- [ ] Changes to necessary documentation made
- [ ] Has test cases for code.

# Tests Performed:

- [ ] Upload data to Datastore
- [ ] Retreive data from Datastore
- [ ] Map Visualization 
- [ ] Network Map Visualization 
- [ ] List other manual test cases performed

### CODE REVIEW

File: src/main/webapp/js/table.js [hunk 1]
```diff
@@ -3,16 +3,17 @@ $(document).ready(function () {
...
-        url: '/data', //table.json for hardcode
+        url: '/data',
         success: function (data) {
             for (i in data) {
+
                 var table = document.getElementById("table");
                 var row = document.createElement("tr");
 
-                if (data[i].flagged == true) {
+                if (data[i].flagged === "TRUE") { //change for string value in malicous final
```
Comment (arunk054) [inline src/main/webapp/js/table.js]: always use case insensitive comparisons like `tolower(data[i].flagged) == "true"`

**Review by arunk054 — APPROVED** [review]
Good job on putting together these charts. Would be nice if you could attach screenshots to the PR.

**Comment by Erik-Handeland** [comment]
image

New charts visuals

**Comment by Erik-Handeland** [comment]
Screen Shot 2020-08-10 at 10 30 30 AM
Updated table view

**Pull request merged**